### DISCUSSION TITLE
Options pour empêcher une réservation périodique d'être posée pendant les vacances

### DISCUSSION THREAD

**Discussion by ydeydier2** [JeromeDevome/GRR#314] [Idées]
Bonjour,

GRR est utilisé par un conservatoire de musique (69 salles gérées !)
Les utilisateurs souhaiteraient pouvoir empêcher qu'une réservation **périodique** ne soit "posée" pendant les vacances scolaires. Une option permettrait cela sur le formulaire d'ajout des réservations.
Cela n'empêcherait pas une réservation non périodique d'être posée pendant ces vacances.
Remarque : j'ai déjà vu cette demande sur vos forums.

Je vous remercie.

Yann Deydier

**Comment by ynaessens** [comment]
Bonjour,
il est possible de choisir les jours ordinaires ou spéciaux lors de la réservation dans la v3.5.1, mais il n'est pas prévu de bloquer cette possibilité. Ce qui semble être ce que vous souhaitez.
On peut se pencher sur la question. La difficulté étant de ne pas créer une page trop complexe et éviter des lenteurs indésirables.
Cordialement,
YN
